main.js

```diff
@@ -0,0 +1,21 @@
+const listedLinks = document.querySelectorAll('li')
+for (let listedLink of listedLinks) {
+    listedLinks[0].remove()
+}
```

Comment:
if you are specifying the index, you do not need a for loop. 

Also since you are only removing 1 element, you can use `querySelector`, which also avoids you having to handle an array.